src/components/progress-indicator.js

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+import { makeStyles } from '../utils/make-styles'
+import { CheckBold } from '../icons'
+
+
+const styles = makeStyles({
+  activeBorderColor: { borderColor: '#2242CD' },
+  inactiveBorderColor: { borderColor: '#9E9E9E' },
```

Comment:
@sallkall `#2242cd` and `#9E9E9E` These colours are not in our design colour system. Should we add it to the our colour guide?

Reply:
Oh this is what we talked about during design sync this morning, @vlatawiec and I already fixed it and shared with selin.

Reply:
I heard but I didn't match the content with this PR. My bad, Tyty.